wp-content/plugins/zippicks-geo-service/includes/class-rest-controller.php

```
@@ -0,0 +1,496 @@
+        }
+        
+        return false;
+    }
+    
+    /**
+     * Check rate limiting
+     * 
+     * @param \WP_REST_Request $request
+     * @return bool
+     */
+    private function check_rate_limit($request) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)